wandb/sdk/wandb_init.py

```diff
@@ -180,13 +173,17 @@ def setup(self, kwargs) -> None:  # noqa: C901
         config_include_keys = kwargs.pop("config_include_keys", None)
         config_exclude_keys = kwargs.pop("config_exclude_keys", None)
 
-        # todo: deprecate config_include_keys and config_exclude_keys
-        # if config_include_keys or config_exclude_keys:
-        #     wandb.termwarn(
-        #       "config_include_keys and config_exclude_keys are deprecated:"
-        #       " use config=wandb.helper.parse_config(config_object, include=('key',))"
-        #       " or config=wandb.helper.parse_config(config_object, exclude=('key',))"
-        #     )
+        deprecated_kwargs = {
+            "config_include_keys": (
+                "Use `config=wandb.helper.parse_config(config_object, include=('key',))` instead."
+            ),
+            "config_exclude_keys": (
+                "Use `config=wandb.helper.parse_config(config_object, exclude=('key',))` instead."
+            ),
+        }
+        for deprecated_kwarg, msg in deprecated_kwargs.items():
+            if locals().get(deprecated_kwarg):
```

Comment:
I think there is a problem using `local()`. See this PR where we wrap it in `dict`:
https://github.com/wandb/client/pull/1390

Reply:
oh, totally forgot about that, thank you!!
rewrote this a bit, hopefully it's more clear now. (i was checking if the local `config_<in|ex>clude_keys` variables were not None). Wrote it this way in case we want to deprecate more kwargs in the future.